src/selic-entries.ts

```diff
@@ -0,0 +1,12 @@
+export async function selicEntries(initialDate: string, finalDate: string) {
+    const url = new URL('https://api.bcb.gov.br/dados/serie/bcdata.sgs.11/dados?formato=json');
+    const params = new URLSearchParams([
+        ['dataInicial', initialDate],
+        ['dataFinal', finalDate],
+    ]);
+    url.search = params.toString();
+    
+    const response = await fetch(url.toString());
+    const selic_entries = await response.json();
+    return selic_entries;
```

Comment:
Do we need this variable?

```suggestion
    return await response.json();
```